core/src/main/java/dk/alexandra/fresco/lib/math/integer/exp/ExponentiationOpenExponent.java

```diff
@@ -46,12 +46,11 @@ public ExponentiationOpenExponent(Computation<SInt> x, BigInteger e) {
   @Override
   public Computation<SInt> build(SequentialNumericBuilder builder) {
     if (exponent.equals(BigInteger.ZERO)) {
-      return builder.numeric().known(BigInteger.valueOf(1));
+      throw new RuntimeException("Now why would you exponentiate a SInt with 0?");
```

Comment:
You may want to exponentiate with by a value not known at compile time which could be zero? Of course you could test for that outside of this class and mark it as only compatible with strictly positive exponents. However, in that case this should be an "illegal argument" exception.